backend/src-common/src/main/java/org/eclipse/sw360/datahandler/db/ComponentRepository.java

```diff
@@ -215,6 +229,13 @@ public Set<String> getComponentIdsByName(String name, boolean caseInsenstive) {
         return queryForIdsAsValue("byname", name);
     }
 
+    public Set<String> getComponentIdsByVCS(String vcs, boolean caseInsenstive){
```

Comment:
Why do you have an additional parameter "caseInsensitive" when it is being set to true at all the places it is being called